src/useCases/user/create/create-user.usecase.ts

```diff
@@ -1,48 +1,69 @@
 import { AppError, Report, StatusCode } from '@expressots/core'
 import { provide } from 'inversify-binding-decorators'
-import {
-    ICreateUserRequestDTO,
-    ICreateUserResponseDTO,
-} from './create-user.dto'
+import { CreateUserRequestDTO, CreateUserResponseDTO } from './create-user.dto'
 import { UserRepository } from '@repositories/user/user.repository'
-import { User } from '@entities/user.entity'
+// import { User } from '@entities/user.entity'
+import { genSaltSync, hashSync } from 'bcrypt'
+import { Prisma } from '@prisma/client'
```

Comment:
Remover linha comentada
```suggestion
import { genSaltSync, hashSync } from 'bcrypt'
import { Prisma } from '@prisma/client'
```